salt/modules/file.py

```diff
@@ -3500,7 +3500,7 @@ def path_exists_glob(path):
     Expansion allows usage of ? * and character ranges []. Tilde expansion
     is not supported. Returns True/False.
 
-    .. versionadded:: Hellium
+    .. versionadded:: Helium
```

Comment:
Technically this should be `2014.7`. I probably missed changing this codename reference to the version number because of the misspelling. Mind fixing that?

Reply:
Done